hubverse_annotator/app.py

```diff
@@ -418,6 +429,27 @@ def load_data_ui() -> tuple[pl.DataFrame, pl.DataFrame]:
     return observed_data_table, forecast_table
 
 
+@st.cache_data
+def cached_location_lookup(locs: list[str]) -> pl.DataFrame:
```

Comment:
```suggestion
def cache_location_lookup(locs: list[str]) -> pl.DataFrame:
```

Or some other name. This is a function that does the caching. It is not itself a cached object. In general, functions should be named with verbs.

Reply:
I think just use `forecasttools.location_lookup(location_vector=locs, location_format="abbr")` inline rather than define this one line function.

Reply:
Unless that is key to using the `cache_data` decorator.

Reply:
Resolved by cacheing the extraction of available locations from the forecast table in e7cb808